server/utils/conditionsProvider.ts

```diff
@@ -75,13 +75,19 @@ export function expandAdditionalConditions(conditions: AdditionalCondition[]): A
           value = adjustCase(rules?.case as string, value)
           value = rules?.includeBefore ? `${rules.includeBefore}${value}` : `${value}`
           value = rules?.type === InputTypes.ADDRESS ? formatAddress(value) : value
+          if (rules?.beforeVowel) {
+            value = startsWithAVowel(value) ? `${rules.beforeVowel}${value}` : `${rules.beforeConsonant}${value}`
+          }
           conditionText = replacePlaceholderWithValue(ph, conditionText, value)
         } else {
           // List of values for this placeholder (lists of values can have listType 'AND' or 'OR')
           const rules = inputConfig.find(item => item.name === fieldName)
           let value = produceValueAsFormattedList(rules?.listType as string, matchingDataItems)
           value = adjustCase(rules?.case as string, value)
           value = rules?.includeBefore ? `${rules.includeBefore}${value}` : `${value}`
+          if (rules?.beforeVowel) {
```

Comment:
```suggestion
          if (rules?.handleIndefiniteArticle) {
              value = startsWithAVowel(value) ? `an ${value}` : `a ${value}`
          }
```

Reply:
Yes, seems better so will look to add over the weekend. Thanks.

Reply:
A quick read suggests the handling of indefinite articles should be a bit more complex as there are a number of edge cases, but this will catch the majority. Things like `hourly` or `honourable` should be an `an` word, but don't start with a vowel. However, in this context, we are unlikely to hit many of these exceptions.